network-api/networkapi/wagtailpages/customblocks.py

```diff
@@ -41,6 +41,15 @@ class Meta:
         template = 'wagtailpages/blocks/image_block.html'
 
 
+class AirTableBlock(blocks.StructBlock):
+    url = blocks.URLBlock()
+    height = blocks.IntegerBlock()
```

Comment:
is there a default we can maybe set?

Reply:
Ah, good call; I meant to do that and add a helper description